modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -10519,7 +10519,7 @@ x-moved-a-web-content-to-the-recycle-bin={0} moved a web content to the Recycle
 x-not-found={0} not found.
 x-notes={0} Notes
 x-of-x-products-selected={0} of {1} Products Selected
-x-of-x-selected={0} of {1} Selected
+x-of-x-selected={0} of {1} selected
```

Comment:
@tinycarol We have a very strict convention when it comes to format of labels in UI. It can be one of two things:
1. A title. In this format, all words are capitalized. Well, most words, English is [unnecessarily complicated around this.](https://en.wikipedia.org/wiki/Capitalization_in_English)
2. A sentence. In this format, words are not capitalized, and label must end with a dot.

You can see this convention in all surrounding labels.

With this in mind, `x-of-x-selected` can be either `{0} of {1} Selected` or `{0} of {1} selected.` The same is true for `x-results-for-x`. A form of sentence looks better to me, I'd say this PR is only missing two dots, but I'm not a native speaker :slightly_smiling_face:  /cc @john-co

If we are doing this, we should also change `x-of-x-products-selected`, to be consistent. It is used on Global menu > Commerce > Products.

Reply:
Hey guys, IMHO we shouldn't ask for prs to be closed without the knowledge of the author, specially if they're learning the processes :)

About the keys pattern, do we have any document explaining it? It would be helpful for everyone (QA, devs and design) to follow the same format.

Reply:
Forwarding then! Wish me luck 🤞